test/Microsoft.AspNet.Mvc.TagHelpers.Test/LinkTagHelperTest.cs

```diff
@@ -197,16 +192,6 @@ public static TheoryData RunsWhenRequiredAttributesArePresent_Data
                     {
                         new TagHelperAttributeList
                         {
-                            ["asp-append-version"] = "true"
-                        },
-                        tagHelper =>
-                        {
-                            tagHelper.AppendVersion = true;
-                        }
```

Comment:
This is covered in `RendersLinkTags_AddsFileVersion` test and a similar one in `SciptTagHelperTest`.